core/storage/suggestion/gae_models.py

```diff
@@ -3506,6 +3506,26 @@ class TranslationCoordinatorsModel(base_models.BaseModel):
     coordinators_count = datastore_services.IntegerProperty(
         indexed=True, required=True)
 
+    # Here we use MyPy ignore because the signature of this method
+    # doesn't match with BaseModel.get().
+    # https://mypy.readthedocs.io/en/stable/error_code_list.html#check-validity-of-overrides-override
+    @classmethod
+    def get( # type: ignore[override]
+        cls, language_code: str
+    ) -> Optional[TranslationCoordinatorsModel]:
+        """Gets the TranslationCoordinatorsModel
+        matching the supplied language_code.
+
+        Args:
+            language_code: str. ISO 639-1 language code.
+
+        Returns:
+            TranslationCoordinatorsModel|None. The matching
+            TranslationCoordinatorsModel, or None if no
+            such model instance exists.
+        """
+        return cls.get_by_id(language_code)
+
```

Comment:
Why do you even need to implement this? `get` is implemented in the base model

Reply:
I didn't knew this. Fixed it. Thanks for the information.

Reply:
You can do `.get(id, strict=False)` and that will return either the model or `None`, you do not need this reimplementation.

Reply:
Removed it. Thanks for the information. PTAL again.